test/routes/src/server/routes/t_launchpad.js

```diff
@@ -8,7 +8,7 @@ import {
   resetMemcached,
   setupInMemoryMemcached
 } from '../../../../../src/server/helpers/memcached';
-import getSnapNameCacheId from '../../../../../src/server/helpers/github';
+import { getSnapNameCacheId } from '../../../../../src/server/handlers/github';
```

Comment:
Why other tests that use this didn't fail when it was undefined? ... it's a mystery.